pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -73,11 +91,16 @@
 # Instrument test attributes
 
 _test_dates = {'': {'vtec': dt.datetime(2017, 11, 19),
-                    'site': dt.datetime(2001, 1, 1)}}
-
+                    'site': dt.datetime(2001, 1, 1),
+                    'los': dt.datetime(2023, 1, 1)}}
+_test_load_opt = {'': {'los': [{'los_method': 'site', 'los_value': 'zzon',
+                                'gnss_network': 'glonass'},
+                               {'los_method': 'time',
+                                'los_value': dt.datetime(2023, 1, 1)}]}}
+_test_download_ci = {'': {'los': False}}  # Download is too large to test
```

Comment:
```suggestion
_password_req = {'': {'los': True}}  # Download is too large to test
```
Since tests are not expected to work either locally or on the CI, disable both

Reply:
still needs to be fixed